src/test/resources/application-test.yml

```diff
@@ -66,12 +66,6 @@ hmpps.sqs:
       queueName: "adjudication-${random.uuid}"
       dlqName: "adjudication-dlq-${random.uuid}"
       subscribeTopicId: domainevents
-      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\", \"adjudication.damages.updated\", \"adjudication.evidence.updated\", \"adjudication.hearing.created\",\
```

Comment:
even less  no value when running tests other then a headache keeping them up to date

Reply:
We don't have any tests that fire an event not in that list and assert we never received it